plugins/templates/create.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const hoek = require('hoek');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/templates',
```

Comment:
@FenrirUnbound  Yeah. I'm open to having that as well. We need to think more about it. We might even need to change the naming convention for templates, since currently we allow `/` in the name. But I would say it doesn't belong to the scope of this PR. This PR provides the base `POST` to create, and `GET /id` to get which might be helpful in some cases (and we do have that consistently for other models). 

But we do need to implement the `/templates/name/versions/version`, or `templates?name=name&version=version`, or whichever way we decide to implement.